### ISSUE TITLE
VsCode.dev: cannot backspace to erase selected text, neither erase newline

### ISSUE THREAD

**Issue by lattice0** [microsoft/vscode#202787]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version:  1.85.2 (vscode.dev web version)
- OS Version: Android 14, One UI 6.1, Brave Browser v1.61.120, **happens on Android 14 Chrome as well (120.0.5099.144)**

Steps to Reproduce:

1. Open any code in vscode.dev
2. Try to backspace to delete a selected text, or backspace on a newline to remove the newline and go back to the previous line
3. Nothing happens

Selecting some code and doing backspace to erase does not work, but typing a letter or space to substitute it by text works, then I can remove the letter by backspace.

My solution: move from "computer version" to "mobile version" in the browser. But shouldn't it work in both?

**Comment by zaydzuhri** [comment]
Oh my god thank you for writing your solution. I was losing my mind over this. But even in "mobile version", you have to press backspace multiple times before it starts deleting newlines. This should be urgently fixed.

**Comment by lattice0** [comment]
@zaydzuhri yes, Microsoft should pay attention to tablets/samsung dex mode as they are the perfect application for vscode web

**Comment by stavros-tsioulis** [comment]
For me the entire site crashes instead (Galaxy Tab S11 Ultra, Chrome 141). Unfortunately vscode.dev is unusable in DeX mode at this moment unless there is some other workaround.
